packages/clerk-js/src/utils/passwords/password.ts

```diff
@@ -65,9 +65,10 @@ export const createValidatePassword = (config: UsePasswordConfig, callbacks?: Va
       });
     }
 
-    internalOnValidation({
-      ...result,
-      complexity: failedValidationsComplexity,
-    });
+    if (result.complexity && Object.keys(result.complexity).length === 0 && show_zxcvbn) {
+      return;
+    }
+
+    internalOnValidation(result);
```

Comment:
@desiprisg Why don't we want to  include `complexity: failedValidationsComplexity` here ?

Reply:
It's already merged into the object above, so it was unnecessary.